### PR TITLE
[LEVWEB-1393] Added data redaction info

### PR DESCRIPTION
**Pull request by aarongill1**
...for API spec / docs

### CODE REVIEW

File: assets/sections/life-event-verification-lev-api/templates/guides/ViewableData.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+Viewable Data
+========
+
+The data any user can view will be dependent on the permissions of the user, regardless of whether we hold the data.
```
Comment (daniel-ac-martin) [inline assets/sections/life-event-verification-lev-api/templates/guides/ViewableData.md:4]: I think we should define the granularity. We need to point out that even if a field / property is marked as required in the JSON schema (i.e. We expect to always have it available) you won't get it unless you have been given permission for it.

Comment (daniel-ac-martin) [reply assets/sections/life-event-verification-lev-api/templates/guides/ViewableData.md:4]: This is a good first draft though.

File: assets/sections/life-event-verification-lev-api/templates/guides/ViewableData.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
...
+
+If you are unable to view any fields that you have permission to access, please email lev-enquiries@digital.homeoffice.gov.uk or raise an issue at https://github.com/UKHomeOffice/lev-api-spec/issues
```
Comment (daniel-ac-martin) [inline assets/sections/life-event-verification-lev-api/templates/guides/ViewableData.md:6]: "If you are unable to view a field you require then please e-mail" ...

Something like that might be better. As maybe the permissions are wrong and need to be adjusted.

I don't think we should point them at GitHub as it's not a code issue.

We should either point them at an e-mail address or to their own service desk.

File: assets/sections/life-event-verification-lev-api/templates/reference/death-records.md [hunk 3]
```diff
@@ -2,3 +2,7 @@ Overlay: true
 
 [[description]]
 Look up and search for death records. Individual death records can be looked up using the `systemNumber` printed in the bottom-left of the death certificate. Alternatively, it is possible to search for death records by supplying the surname, at least the first forename and either the date of birth or the date of death of the deceased.
+
+Note: [You will only receive the fields you are authorized to view][viewableData], regardless of whether we hold them on file.
```
Comment (daniel-ac-martin) [inline assets/sections/life-event-verification-lev-api/templates/reference/death-records.md:6]: "regardless of whether we hold them on file and the [JSON schema]."

And we should be able to link to the schema for Death-v1.

**Comment by daniel-ac-martin** [comment]
Looks pretty good. I just think we need to tweak the content a bit.

Might be good to get Nigel to take a look at this. (Although we might need to deploy a draft first so he can see it in context.)

File: assets/sections/life-event-verification-lev-api/templates/guides/ViewableData.md (hunk 2 above)
Comment (aarongill1) [reply assets/sections/life-event-verification-lev-api/templates/guides/ViewableData.md:6]: Changed

File: assets/sections/life-event-verification-lev-api/templates/reference/death-records.md (hunk 3 above)
Comment (aarongill1) [reply assets/sections/life-event-verification-lev-api/templates/reference/death-records.md:6]: Done

File: assets/sections/life-event-verification-lev-api/templates/guides/ViewableData.md (hunk 1 above)
Comment (aarongill1) [reply assets/sections/life-event-verification-lev-api/templates/guides/ViewableData.md]: Done

File: .deps/package.json [hunk 4]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "lev-api-spec",
-  "version": "0.5.0",
+  "version": "0.6.0",
```
Comment (daniel-ac-martin) [inline .deps/package.json:3]: Apologies, it looks like we've been neglecting to update this with the tags on the repo.

I think this needs to be v0.13.0.

**Comment by daniel-ac-martin** [comment]
Looks good otherwise.

File: .deps/package.json (hunk 4 above)
Comment (aarongill1) [reply .deps/package.json]: Updated

**Pull request merged**